test/metric_value_benchmark/net_test.go

```diff
@@ -44,6 +44,10 @@ func (m *NetTestRunner) getAgentRunDuration() time.Duration {
 	return minimumAgentRuntime
 }
 
+func (m *NetTestRunner) setupBeforeAgentRun() error {
```

Comment:
nit: I think you can have the function return nil in `BaseTestRunner`, so you don't need to implement it for each of these runners. 

Super quick example: https://go.dev/play/p/EjvuJWSpJ_n